Sources/PluginLibrary/Descriptor.swift

```diff
@@ -285,6 +286,9 @@ public final class EnumValueDescriptor {
     self.index = index
     self.enumType = enumType
     aliasOf = aliasing
+
+    let fullName = "\(enumType.fullName).\(proto.name)"
```

Comment:
I've been trying to keep `lazy` to a minimum in here pending some performance analysis on generation, as there is an easy way to thread the plugin and generate files in parallel, but all of the lazy usage would have to go.